docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+db:
+    restart: always
+    image: mysql:5.6
+    ports:
+        - "3306:3306"
+    environment:
+        MYSQL_ROOT_PASSWORD: changeme
+        #the values below should match the credentials used in 
+        #instance/local_settings.py
+        MYSQL_USER: nbsap
+        MYSQL_PASSWORD: nbsap
+        MYSQL_DATABASE: nbsap
+web:
+    build: .
+    command:
+        python instance/manage.py runserver 0.0.0.0:8000
+    volumes:
+        - .:/nbsap
+    ports:
+        - "8000:8000"
+    links:
+        - db
+    environment:
+        - DEBUG=True
+        - DB_NAME='nbsap'
+        - DB_USER='nbsap'
+        - DB_PASSWORD='nbsap'
+        - DB_HOST='db'
+        - SENTRY_DSN='http://e58d6f13a36d419d81c1d1e44e5e2519:d67d512c5f4343c0aebcdca30bc64871@sentry.mojito.edw.ro/16'
+        - EU_STRATEGY=True
+        - EU_STRATEGY_ADD=True
+        - NAT_STRATEGY=False
+        - SITE_HEADER='NBSAP'
+        - INFO_HEADER=False
```

Comment:
It's better practice to use an environment file for such amount of environment variables, please see: https://docs.docker.com/compose/yml/#env-file